### PR TITLE
docs:  rust examples minor fixes

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

Some minor improvements to the rust examples

## Links to any relevant issues

part of  #388 

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: sdk/examples/client/participation.rs [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 //! This example shows how to participate in voting events.
 //!
-//! Command to create an event:
+//! Command to create an event, when you your node is located on the same machine:
```
Comment (Thoralf-M) [inline sdk/examples/client/participation.rs]:
```suggestion
//! Command to create an event, when your node is located on the same machine:
```

File: sdk/examples/wallet/offline_signing/2_sign_transaction.rs [hunk 2]
```diff
@@ -68,25 +50,20 @@ async fn main() -> Result<()> {
...
-) -> Result<PreparedTransactionData> {
+async fn read_prepared_transaction_from_file() -> Result<PreparedTransactionData> {
     use tokio::io::AsyncReadExt;
 
     let mut file = tokio::io::BufReader::new(tokio::fs::File::open(PREPARED_TRANSACTION_FILE_PATH).await?);
     let mut json = String::new();
     file.read_to_string(&mut json).await?;
 
-    Ok(PreparedTransactionData::try_from_dto(
+    Ok(PreparedTransactionData::try_from_dto_unverified(
```
Comment (Thoralf-M) [inline sdk/examples/wallet/offline_signing/2_sign_transaction.rs:67]: Good change, forgot to do it when `try_from_dto_unverified` got added

File: sdk/examples/wallet/offline_signing/2_sign_transaction.rs [hunk 3]
```diff
@@ -68,25 +50,20 @@ async fn main() -> Result<()> {
         inputs_data: prepared_transaction_data.inputs_data,
     };
 
-    println!("Signed transaction.");
```
Comment (Thoralf-M) [inline sdk/examples/wallet/offline_signing/2_sign_transaction.rs]: Isn't it good to see this output?

Comment (qrayven) [reply sdk/examples/wallet/offline_signing/2_sign_transaction.rs]: It's good. Just deleted too much while deleting debug prints"

File: sdk/examples/client/03_simple_block.rs
Comment (DaughterOfMars) [inline sdk/examples/client/03_simple_block.rs:1]: I missed why this was deleted

Comment (qrayven) [reply sdk/examples/client/03_simple_block.rs:1]: It was just a copy of this:
https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/client/block/00_block_no_payload.rs

**Review by DaughterOfMars — DISMISSED** [review]

File: sdk/examples/wallet/offline_signing/3_send_transaction.rs [hunk 4]
```diff
@@ -20,12 +21,22 @@ use iota_sdk::{
...
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    // This example uses secrets in environment variables for simplicity which should not be done in production.
+    dotenvy::dotenv().ok();
+
+    // Setup Stronghold secret_manager
+    let secret_manager = StrongholdSecretManager::builder()
+        .password(std::env::var("STRONGHOLD_PASSWORD").unwrap())
+        .build(STRONGHOLD_SNAPSHOT_PATH)?;
```
Comment (Thoralf-M) [inline sdk/examples/wallet/offline_signing/3_send_transaction.rs]: This shouldn't be required, the "online part" shouldn't have a secret manager

Comment (qrayven) [reply sdk/examples/wallet/offline_signing/3_send_transaction.rs]: yaeah, that's a good point. I was thinking, that we cannot create a wallet without a secret manager. But apparently, we have `SecretManager::Placeholder` for this. Fixed

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**